.drone.yml

```diff
@@ -467,7 +467,7 @@ volumes:
 
 kind: pipeline
 type: exec
-name: push-build-darwin-amd64
+name: push-build-native-darwin-amd64
```

Comment:
What does "native" vs "non-native" mean?

Reply:
I used `native` in the Teleport Connect windows build to indicate it was ruin natively in Windows, rather than cross-compiled on  linux. Was that the intent here as well @fheinecke ?

Reply:
Originally I was just following what Connect did without really understanding it. I've removed "native" as it's not needed here.